packages/cozy-external-bridge/src/container/useListenBridgeRequests.ts

```diff
@@ -54,9 +54,10 @@ export const useListenBridgeRequests = (
           .collection('io.cozy.files')
           // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
           .createFile('Empty content', {
```

Comment:
@paultranvan Should we change this 'Empty content' text ?

Reply:
I think we tried empty string and it wasn't working right? I don't recall precisely why?

Reply:
Oh you're right, cozy-client checks with a simple if(content) so empty string is falsy...